### PR TITLE
Successfull logout

### PR DESCRIPTION
**Pull request by GabrielAdward**
# Pull Request

## Task list 
- [x] Checked the code structure so it follows guide lines
- [x] Checked that all the points in the Issue are solved
- [x] Checked that all the old tests still got approved

## Added tests
- [ ] Yes, I have added tests
- [x] No, I have not added tests
- [ ] No, I need help with adding tests

### CODE REVIEW

**Comment by GabrielAdward** [comment]
If you change to an invalid URL then this will appear 
image

**Review by HeWhoThreadsLightly — COMMENTED** [review]
Looks good, test pass on my local machine.

I would like a new test that checks that user data cant be read after logging out, but we dont have time for it before friday.

**Comment by HeWhoThreadsLightly** [comment]
I would like a test that:
 1 logs in
 2 tries to reed some private user data and checks that it was read succsessfully
 2 logs out
 3 tries to read the same private user data and checkt that it fails to read

**Comment by HeWhoThreadsLightly** [comment]
Strange that the automatic tests have't activated.

File: src/test/logout.test.tsx [hunk 1]
```diff
@@ -0,0 +1,17 @@
+import '@testing-library/jest-dom'
+import AuthService from '../components/AuthService';
+
+
+test("Login, check data, logout and check if information is gone", async () => {
```
Comment (HeWhoThreadsLightly) [inline src/test/logout.test.tsx:5]: Test looks good.

However you need to merge in the canges from main into your branch and solv the merge conflicts.
checkout main
pull
checkout successfull_logout
git merge main (or use tortoise git to merge)
fix merge conflicts (files with mexed content from both branches)
git push

**Review by HeWhoThreadsLightly — CHANGES_REQUESTED** [review]

**Pull request merged**